modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/add_to_cart/AddToCartButton.js

```diff
@@ -58,60 +128,29 @@ function AddToCartButton({
 			disabled={disabled}
 			displayType="primary"
 			monospaced={settings.iconOnly && settings.inline}
-			onClick={(event) => {
```

Comment:
why do you need to move this logic?

Reply:
I moved this logic outside the onClick function because I need to call it to add in cart after the modal is open.